index.css

```diff
@@ -91,7 +91,7 @@ main {
     font-weight: 400;
 }
 
-.order-summary {
+#order-summary {
```

Comment:
REFRAIN FROM USING IDs IN THE CSS. A hierarchy battle can develop. That's why I added classes on elements in the HTML you put IDs on. Those classes have the same names as the IDs and I kept the IDs so you can refer to them in the JS.